dev/com.ibm.ws.grpc.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,49 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.grpc.client/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+
+clientConfig=GRPC Client Properties
+clientConfig.desc=Configuration properties to be applied to gRPC targets that match the specified URI when they are constructed
+
+target=Remote target
+target.desc=The remote gRPC target URI, with wildcard support.
+
+headersToPropagate=HTTP header names to propagate
+headersToPropagate.desc=HTTP header names to propagate from an inbound request to an outbound call.
+
+clientInterceptors=GRPC client Interceptors
+clientInterceptors.desc=A list of fully qualified class names for gRPC client interceptor classes.
+
+authnToken=Authorization token
+authnToken.desc=The type of authorization token to propagate.  Currently this must be "jwt" or "oauth".
+
+enableKeepAlive=Keep alive
+enableKeepAlive.desc=Enable grpc keep alive.
+
+keepAliveTime=Keep alive interval
+keepAliveTime.desc=The time to wait for new messages before sending a new keep alive ping.
+
+keepAliveTimeout=Keep alive timeout
+keepAliveTimeout.desc=End the connection if a keepalive response is not received within this time.
+
+maxInboundMessageSize=Maximum message size
+maxInboundMessageSize.desc=The maximum inbound message size.
+ 
+sslConfig=The id of an ssl reference in server.xml
+sslConfig.desc=An id of an ssl reference found in server.xml.  That ssl configuration specified by that reference is used.
+ 
+other=Any other variables
+other.desc=Any other variables can also be specified and will be passed to the GrpcTarget intact. 
```

Comment:
avoid future tense.

Any other variables that can  be specified and passed to the GrpcTarget intact.